pkgs/tools/package-management/dpkg/default.nix

```diff
@@ -1,5 +1,19 @@
-{ lib, stdenv, fetchgit, perl, gnutar, zlib, bzip2, xz, zstd
-, libmd, makeWrapper, coreutils, autoreconfHook, pkg-config
+{ lib
+, stdenv
+, fetchgit
+, perl
+, gnutar
+, zlib
+, bzip2
+, xz
+, zstd
+, libmd
+, makeWrapper
+, coreutils
+, autoreconfHook
+, pkg-config
+, diffutils
+, glibc
```

Comment:
Adding `glibc` here breaks darwin builds, where glibc is not supported.

Reply:
Another day, another Darwin quirk to be fixed.
It deserves another issue.

Reply:
Fix: https://github.com/NixOS/nixpkgs/pull/250931